src/content/consul/install-landing.json

```
@@ -13,31 +13,14 @@
 		{
 			"divider": true
 		},
+		{ "heading": "Kubernetes Installs" },
 		{
-			"heading": "Get Started"
-		},
-		{
-			"title": "What is Consul?",
-			"fullPath": "/consul/docs/intro"
-		},
-		{
-			"title": "Consul with Kubernetes",
-			"fullPath": "/consul/tutorials/get-started-kubernetes/kubernetes-gs-deploy"
-		},
-		{
-			"title": "Consul with Virtual Machines",
-			"fullPath": "/consul/tutorials/get-started-vms/virtual-machine-gs-deploy"
-		},
-		{
-			"title": "HCP Consul",
-			"fullPath": "/consul/tutorials/get-started-hcp/hcp-gs-deploy"
-		},
-		{
-			"divider": true
+			"title": "Consul K8s CLI",
```

```suggestion
			"title": "Consul Kubernetes CLI",
```

`K8s` was in the original, but seems to me a bit undiscoverable/informal vs `Kubernetes`.